substrings.rb

```diff
@@ -0,0 +1,8 @@
+def substrings(string, dictionary)
```

Comment:
I know this is how it's written in the assignment - but `dictonatry` is not a good name - it doesn't tell the user what to send
```suggestion
def substrings(string, substrings)
```